_Week9/Group4/estimation.do

```diff
@@ -0,0 +1,99 @@
+********************************************************************************
+* ECON 491
+* Week 9 - Estimation
+* Group 4
+* November 14, 2022
+********************************************************************************
+
+cap prog drop projectdata_sim
+
+prog define projectdata_sim, rclass
+
+clear
+
+********************************************************************************
+* setting up inputs
+********************************************************************************
+
+{
+	
+syntax anything
+local bias: word 1 of `anything' //a dummy for whether the results are biased or not
```

Comment:
Very, very small technical point (your code is great and very readable so I am grasping at straws!)--I believe whether or not it was biased is supposed to be an option, not an input (e.g., could be included after a comma to specify a biased estimate, or omitted entirely to specify an unbiased estimate).

Reply:
Fix looks good!